_im-plugin/ism/policies.md

```diff
@@ -162,6 +163,69 @@ Parameter | Description | Type | Required
 
 For information about setting replicas, see [Primary and replica shards]({{site.url}}{{site.baseurl}}/opensearch#primary-and-replica-shards).
 
+### shrink
+
+Allows you to reduce the number of primary shards in your indexes. With this action, you can specify:
+
+- The number of primary shards that the target index should contain.
+- A max shard size for the primary shards in the target index.
+- Specify a percentage to shrink the number of primary shards in the target index.
+
+```json
+"shrink": {
+    "num_new_shards": {
```

Comment:
Based on the other actions, it looks like this is meant to be an example but there are some issues with it. Here is a valid replacement:
```
"shrink": {
    "num_new_shards": 1,
    "target_index_name_template": {
        "source": "{{ctx.index}}_shrunken"
    },
    "aliases": [
       "my-alias": {}
    ],
    "force_unsafe": false
}
```

Reply:
I understand. I thought with the other example I would attempt to get all the options in, but in retrospect this makes more sense. :)